libs/components/split-view/src/lib/modules/split-view/split-view-workspace.component.ts

```diff
@@ -1,22 +1,11 @@
 import {
```

Comment:
👀 👇 Split view saw the greatest number of changes since it was doing some fairly "boutique" breakpoint handling.